tvbwidgets/tests/test_drive_widget.py

```diff
@@ -0,0 +1,122 @@
+import os
```

Comment:
Minor, but: It would be good to keep the project header also in the test file.

Reply:
Thanks, I missed that. Will add it soon.